pkg/cmd/service/service.go

```diff
@@ -7,14 +7,15 @@ import (
 	"log"
 	"time"
 
+	grpc_auth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
```

Comment:
Wired up the `grpc_auth` middleware package in favor of a vanilla grpc interceptor because this allows projects already established in the grpc middleware ecosystem to plug and play their existing `grpc_auth.AuthFunc` with us.

Reply:
This is good 👍🏽 

But please try to keep PRs within scope, this could have been a separate one.

Reply:
Sounds good.

I felt this approach within the scope of this PR made the most sense because it made the work a little easier for me, and so although it was a tad tangential, it made it easier for me to integrate the authless health checks within the scope of this PR.